charts/linkerd2-cni/templates/cni-plugin.yaml

```diff
@@ -143,8 +143,12 @@ data:
         "outgoing-proxy-port": {{.Values.outboundProxyPort}},
         "proxy-uid": {{.Values.proxyUID}},
         "ports-to-redirect": [{{.Values.portsToRedirect}}],
-        "inbound-ports-to-ignore": [{{.Values.ignoreInboundPorts}}],
-        "outbound-ports-to-ignore": [{{.Values.ignoreOutboundPorts}}],
+        "inbound-ports-to-ignore": [
+          {{- include "util.splitStringList" .Values.ignoreInboundPorts -}}
+        ],
+        "outbound-ports-to-ignore": [
+          {{- include "util.splitStringList" .Values.ignoreOutboundPorts -}}
+        ],
```

Comment:
These includes need to be updated, in light of the util->helper recent change

Reply:
Actually my problem was elsewhere... the requirements file name is misspelled, and it's missing the `dependencies:` line. Even so, it might be better to replace this `util` prefix with `partials`...

Reply:
Thanks! Also needed to `dep up` the cni chart in `helm-build` apparently otherwise it fails